prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -954,5 +954,75 @@ public async Task<IEnumerable<string>> GetEnrolleeEmails(BulkEmailType bulkEmail
                 .DecompileAsync()
                 .ToListAsync();
         }
+
+        public async Task<EnrolleeAbsence> CreateEnrolleeAbsenceAsync(int enrolleeId, DateTime startTimestamp, DateTime endTimestamp)
+        {
+            if (startTimestamp >= endTimestamp)
+            {
+                throw new ArgumentException($"endTimestamp is not after the startTimestamp of absence.");
+            }
```

Comment:
This validation can be on the view model, and we can automatically return a 400 rather expecting to throw here. Search the repo for `AbstractValidator` for some examples (we've only added it to newer VMs).

Reply:
Looked at one and put one in that should do what i want, but only way to check is using a postman endpoint with token so can test shortly.